app/pdfs/print_record_pdf.rb

```diff
@@ -5,15 +5,15 @@ class PrintRecordPdf < Prawn::Document
   def initialize(record)
     super()
     @record = record
```

Comment:
In the same way that `unavailability_calendar` does, by passing it as a method argument